selfdrive/controls/controlsd.py

```diff
@@ -569,9 +569,11 @@ def state_control(self, CS):
 
     CC = car.CarControl.new_message()
     CC.enabled = self.enabled
+
     # Check which actuators can be enabled
+    min_lat_active_speed = self.CP.minSteerSpeed if self.joystick_mode else max(self.CP.minSteerSpeed, MIN_LATERAL_CONTROL_SPEED)
     CC.latActive = self.active and not CS.steerFaultTemporary and not CS.steerFaultPermanent and \
-                   CS.vEgo > self.CP.minSteerSpeed and not CS.standstill
+                   CS.vEgo > min_lat_active_speed and not CS.standstill
```

Comment:
joystick never really supported steering at 0, since 0 is not > 0, but I get it can be useful. Revert the joystick exception in this PR and do separately?

Reply:
Ah, I thought `minSteerSpeed` defaulted to -1 like `minEnableSpeed`.

Reply:
Oh true true, the standstill check actually enforces this